app/views/tasks/projects/data/index.html.erb

```diff
@@ -7,6 +7,9 @@
   <div class="panel content">
     <h3> Export SQL </h3>
     <p><em>Generate a downloadable copy of the database (PostgreSQL dump) with all data referenced in this project. Includes Community data like Sources, People, and Repositories.</em></p>
+
+    <p><em>All exported user passwords are set to</em> 'taxonworks'.</p>
```

Comment:
Because of this (that I totally forgot about) then would be nice to add a warning that it is not a good idea at all to build a public TW instance out of the generated dump.